data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -798,6 +798,68 @@
   ]
 },{
   "type" : "recipe",
+  "result": "mutagen_lamia",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_MUTAGEN",
+  "skill_used": "cooking",
+  "skills_required": [ "firstaid", 1 ],
+  "difficulty": 9,
+  "time": 10000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "recipe_maiar", 8 ]],
+  "qualities":[
+    {"id":"CHEM","level":2,"amount":1}
+  ], "tools": [
+    [
+      [ "hotplate", 25 ],
+      [ "toolset", 25 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "mutagen_slime", 1 ],
+      [ "mutagen_lizard", 1],
+      [ "mutagen_bird", 1]
+    ],
+    [
+      [ "cooking_oil", 1 ],
+      [ "chitin_piece", 4 ]
```

Comment:
No, I have cooking oil as a placeholder (and a cheeky reference to snake oil)there for a future PR, where I add in Snakeskin and other snakeskin related items. I'm not going to address it here, because I haven't set-out to make that kind of stuff in this PR.